docs/source/workflows/predictor_evaluation_workflows.rst

```diff
@@ -0,0 +1,264 @@
+Predictor Evaluation Workflows
+==============================
+
+A :class:`~citrine.informatics.workflows.predictor_evaluation_workflow.PredictorEvaluationWorkflow` evaluates the performance of a :doc:`Predictor <predictors>`.
+Each workflow is composed of one or more :class:`PredictorEvaluators <citrine.informatics.predictor_evaluator.PredictorEvaluator>`.
+
+Predictor evaluators
+--------------------
+
+A predictor evaluator defines a method to evaluate a predictor and any relevant configuration, e.g. k-fold cross-validation evaluation that specifies 3 folds.
+Minimally, each predictor evaluator specifies a name, a set of predictor responses to evaluate and a set of metrics to compute for each response.
+Evaluator names must be unique within a single workflow (more on that `below <#execution-and-results>`__).
+Responses are specified as a set of strings, where each string corresponds to a descriptor key of a predictor output.
+Metrics are specified as a set of :class:`PredictorEvaluationMetrics <citrine.informatics.predictor_evaluation_metrics.PredictorEvaluationMetric>`.
+When defining an evaluator, the top-level metrics should be the union of all metrics computed across all responses.
+The evaluator will only compute the subset of metrics valid for each response.
+
+Cross-validation evaluator
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A :class:`~citrine.informatics.predictor_evaluator.CrossValidationEvaluator` performs k-fold cross-validation on a predictor.
+In addition to a name, set of responses to validate and metrics to compute, this evaluator defines the number of folds, number of trials and set of descriptor keys to ignore when grouping.
+The latter allows candidates with different values for the specified (ignored) keys and identical values all other predictor inputs to be placed in the same fold.
```

Comment:
```suggestion
The latter allows candidates with different values for the specified (ignored) keys and identical values for all other predictor inputs to be placed in the same fold.
```

Reply:
It's a bit of an abstract concept, so I think we need to illustrate _why_ this is something you might want to do.